dagql/backends/camflow/__init__.py

```diff
@@ -0,0 +1,120 @@
+import io
+import json
+import logging
+import subprocess
+from collections import defaultdict, deque
+
+from dagql.backend import Backend, Node, Edge, TraversalOrder
+
+
+NAMESPACE = b'dagql'
+START_DIRECTIVE = NAMESPACE + b': #start'
+NAMESPACE_PREFIX = NAMESPACE + b': '
+
+
+class CamFlowNode(Node):
+    """DAGQL node representation"""
+    def __init__(self, attrdict: dict):
+        self.attrs = attrdict
+
+    def __getitem__(self, attr: str):
+        return self.attrs[attr.lower()]
```

Comment:
@CoffeeTableEspresso Can you confirm that attributes in the interface call will always be in lowercase?

Ah, well, this is a little bit more robust since makes it impossible to misuse the interface...

Reply:
@michaelenglo why would we expect the attributes to be in lowercase? The attributes are case insensitive currently, due to the fact that Jerry always normalizes them.

Reply:
Now that I think about it, I forgot to normalize the case in the filesystem backend.

I guess case-sensitivity can be a "it depends on the backend" kind of thing...

Reply:
I'm fine with that